src/render.ts

```diff
@@ -93,9 +93,9 @@ ${preview ? `\nopen({hash: ${JSON.stringify(resolvers.hash)}, eval: (body) => ev
 ${html.unsafe(rewriteHtml(page.body, resolvers))}</main>${renderFooter(page.footer, resolvers, options)}
 </div>
 </body>
-<script>{${html.unsafe(
-    (await rollupClient(getClientPath("dark-init.js"), options.root, path, {resolveImport, minify: true})).trim()
-  )}}</script>
+<script type="module">import{Generators as g}from'${resolveImport(
```

Comment:
I think the old approach of having this in a separate script was cleaner, and meant we could use minification.

Also, why not just add 

```js
import "./dark.js";
```

to `src/client/index.js` rather than having it as another script element?

Reply:
yes this is much better, thanks! (now fixing tests…)